apps/cms/src/app/app.component.less

```diff
@@ -0,0 +1,87 @@
+:host {
+  display: flex;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.app-layout {
+  height: 100vh;
+}
+
+.menu-sidebar {
+  position: relative;
+  z-index: 10;
+  min-height: 100vh;
+  box-shadow: 2px 0 6px rgba(0, 21, 41, 0.35);
+}
+
+.header-trigger {
+  height: 64px;
+  padding: 20px 24px;
+  font-size: 20px;
+  cursor: pointer;
+  transition:
+    all 0.3s,
+    padding 0s;
+}
+
+.trigger:hover {
+  color: #1890ff;
```

Comment:
It would be valuable to establish colour theme, fonts and z-index scale so that we can reuse these values. Shall I create an issue for that?

Reply:
Yes, the theme may be customized, this was all generated by Ant Design. We'll definitely get to that later.

Reply:
#23